apps/mobile/jest.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
   // resolver: '<rootDir>/jest.resolver.js',
   moduleDirectories: [
     '<rootDir>/node_modules',
-    '<rootDir>/../../packages/shared/node_modules'
+    '<rootDir>/../../node_modules'
```

Comment:
Explanation of this: I had let the @hylo/shared dependencies hoist to solve an issue with the jest babel resolution here, but then forget to point the jest config out of the shared dependencies and back to the global ones...